Unix/configeditor/configeditor.cpp

```diff
@@ -372,7 +392,10 @@ int MI_MAIN_CALL main(int argc, const char** argv)
         // If property isn't the one we want, move along
         if (prop != arg1)
         {
-            cout << line << std::endl;
+            if (!fDontEchoFile)
```

Comment:
Also I cannot combine the two if statements as when the 1st if is true then both cout and continue needs to be executed. When the second if is true then cout should not get executed but continue still needs to be executed. I could not think of better way than this, other than possibly using an if, else-if blocks. 
Let me know if an missing something here.